### ISSUE TITLE
perceptual Loss and sync loss cannot be used at the same time during training

### ISSUE THREAD

**Issue by dy0829** [sahilg06/EmoGen#3]
When I train the Wav2Lip model, adding the sync expert loss on top of the perceptual loss causes the training to collapse. I noticed that you also use perceptual loss in your training process, and I'm wondering if you have encountered a similar situation. If so, how did you solve it?

**Comment by sahilg06** [comment]
I didn't encounter such a situation during training. Also, I cannot think of a reason why perceptual loss and sync loss cannot be used simultaneously. However, I did encounter a situation where my training collapsed even before adding perceptual loss. I changed the optimizer's hyperparameters a bit for the same ([here](https://github.com/sahilg06/EmoGen/blob/eb5eba1eac862f0064e6ec08b953c859a9f02b08/train.py#L544)).

**Comment by dy0829** [comment]
Thanks for your reply.  I will check my code again.
